docs/pages/api-reference/liveblocks-react-comments.mdx

```diff
@@ -15,195 +15,217 @@ components to build a commenting experience. Read our
 Comments is currently in private beta and is only available to a select group of
 users. If you want to participate in the beta,
 [sign up for the waitlist](https://forms.gle/xgp54aTMsoH8hrjU9). During this
-beta period, breaking changes may occur, but no data loss. We use beta testers’
-feedback to improve the product. Your input is important and helps us shape the
-final version.
+beta period, breaking changes may occur, but data loss will not. We use beta
+testers’ feedback to improve the product. Your input is important and helps us
+shape the final version.
 
 </Banner>
 
 ## Components
 
-### Customization
-
-#### Styling
+### Thread
 
-The default components come with default styles, import them either at the root
-of your app or directly in CSS depending on your setup.
+Displays a thread of comments. Each comment has a composer for creating replies.
 
-```tsx file="layout.tsx"
-import "@liveblocks/react-comments/styles.css";
+```tsx
+<Thread thread={thread} />
 ```
 
-```tsx file="globals.css"
-@import "@liveblocks/react-comments/styles.css";
-```
+Combine with [`useThreads`](/docs/api-reference/liveblocks-react#useThreads) to
+render a list of the room’s threads and comments.
 
-They are built around a few customizable CSS variables:
+```tsx highlight="10"
+import { Thread } from "@liveblocks/react-comments";
+import { useThreads } from "../liveblocks.config";
 
-```css
-/* Basic */
---lb-radius: 0.5em;
---lb-spacing: 1em;
---lb-accent: #17f;
---lb-accent-foreground: #fff;
---lb-background: #fff;
---lb-foreground: #111;
-
-/* Advanced */
---lb-icon-size: 20px;
---lb-icon-weight: 1.5px;
---lb-avatar-radius: 50%;
---lb-button-radius: calc(0.75 * var(--lb-radius));
---lb-transition-duration: 0.1s;
---lb-transition-easing: cubic-bezier(0.4, 0, 0.2, 1);
---lb-elevation-shadow: 0 0 0 1px rgb(0 0 0 / 4%), 0 2px 6px rgb(0 0 0 / 8%),
-  0 8px 26px rgb(0 0 0 / 12%);
---lb-tooltip-shadow: 0 2px 4px rgb(0 0 0 / 8%), 0 4px 12px rgb(0 0 0 / 12%);
---lb-elevation-background: var(--lb-background);
---lb-elevation-foreground: var(--lb-foreground);
---lb-tooltip-background: #222;
---lb-tooltip-foreground: #fff;
---lb-accent-contrast: 8%;
---lb-foreground-contrast: 6%;
---lb-elevation-foreground-contrast: 6%;
---lb-tooltip-foreground-contrast: 10%;
+function Component() {
+  const { threads } = useThreads();
+
+  return (
+    <>
+      {threads.map((thread) => (
+        <Thread key={thread.id} thread={thread} />
+      ))}
+    </>
+  );
+}
 ```
 
-{/* TODO: Explain automatic color scales (with a palette-type visual) */}
+#### Thread props
```

Comment:
What do you think about naming these just "Props"? Since they're below sections named after the components already.

![](https://github.com/liveblocks/liveblocks/assets/6959425/f9301878-d52a-4eab-8c16-e38530eccc20)

Reply:
Yes!